docs/ref/reference.rst

```diff
@@ -207,6 +209,21 @@ commands can be used, from any node in the setup.
     For details about the options to the command, see above in the ``pg_autoctl
     show events`` command.
 
+  - ``pg_autoctl show uri``
```

Comment:
Please excuse the drive-by comment, but it looks like this might have been mistakenly copied from above. This should say `pg_autoctl show ipaddr`, no?

Reply:
Hi @Bogdanp, you're right, I've updated the PR (rebased on top of current master) and fixed the reference. I'm also going to add a message about the current state of this PR. Thanks!